xml/templates/message_templates/event_online_receipt_text.tpl

```diff
@@ -4,8 +4,13 @@
 {$event.confirm_email_text}
 
 {else}
-Thank you for your participation.  This letter is a confirmation that your registration has been received and your status has been updated to {if $participant_status}{$participant_status}{else}{if $isOnWaitlist}waitlisted{else}registered{/if}{/if}.
-
+{if $participant_status}
+{ts}Thank you for your participation.  This letter is a confirmation that your registration has been received and your status has been updated to {$participant_status}.{/ts}
```

Comment:
should inject the variable like `{ts 1=$participant_status}` since you can't use a variable directly in a translated string.